Source/FlowEditor/Private/Graph/FlowGraphSchema_Actions.cpp

```diff
@@ -197,6 +200,69 @@ UFlowGraphNode* FFlowGraphSchemaAction_NewNode::ImportNode(UEdGraph* ParentGraph
 	return NewGraphNode;
 }
 
+/////////////////////////////////////////////////////
+// New SubNode (AddOn)
```

Comment:
"SubNode" is the editor-side word for "AddOn" (runtime)

Reply:
Yes, but it is quite confusing to understand the editor code, as naming differs from the `AddOn` term. We also have SubGraphs in this plugin.

I would love to rename all "SubNode" occurrences to "AddOn" or "NodeAddOn". I'm only wondering if there would be some naming conflicts with other symbols. I don't think so, but might not see it whole picture yet ;)

Reply:
There's a number of instances in the UE code where there are multiple different types of SubNodes (in Behavior trees, for instance, there are Decorators and Services).  

I primarily am concerned with keeping the Editor-side code clear, so that we don't have "EdNodeNodes" and so on.  That's why i rebranded with AddOns for the runtime.  

But there is some value in keeping the Editor-side generalized to the idea that there may be other types of subnodes in the future, beyond AddOns, that would have a different runtime name.

Am not strongly opinionated on this though.  Something like "NodeAddOn" is fine editor-side, so long as the code reads clearly.

Let's let the code settle a bit before making those changes though.  Let me at least merge back the latest from Flow plugin before any renaming, so I don't have a crazy merge.